tensorboard/plugins/profile/overview_page/overview-page.html

```diff
@@ -0,0 +1,353 @@
+     },
+
+     /* Plots a graph of step time (in ms) against step number */
+     _showDeviceStepChart : function(deviceJson) {
+       var computeTimeSeries = [];
+       var stepTimeSeries = [];
+       var maxStepTime = 0;
+       var step = -1;
+       deviceJson.rows.forEach(function(row, index) {
+         step = step > 0 ? ++step : Number(row.c[0].v);
+         computeTimeSeries.push({"scalar": row.c[1].v,
+				 "step": step,
+				 "tpu_step": Number(row.c[0].v),
+				 "low_watermark": 0})
+         stepTimeSeries.push({"scalar": row.c[1].v + row.c[2].v,
+                              "step": step,
+                              "low_watermark": row.c[1].v})
+         maxStepTime = Math.max(maxStepTime, row.c[1].v + row.c[2].v)
+       })
+       var chart = this.$.device_step_chart;
+       if (chart) {
+         chart.setVisibleSeries(['compute-time', 'step-time = input-time + compute-time']);
+	 chart.setSeriesData('compute-time', computeTimeSeries);
+         chart.setSeriesData('step-time = input-time + compute-time', stepTimeSeries);
+         chart.defaultYRange = [0, maxStepTime * 1.1]
+         chart.smoothingEnabled = false;
+         chart.tooltipColumns = [
+           { title: 'Name', evaluate: (d) => d.dataset.metadata().name },
+           { title: 'Time(ms)', evaluate: (d) => (d.datum.scalar).toFixed(2) },
+           { title: 'Step', evaluate: (d) => d.datum.step }];
+         chart.fillArea = {
+           higherAccessor: (d) => d.scalar,
+           lowerAccessor: (d) => d.low_watermark,
+         };
+         chart.xAxisFormatter = d3.format('d');
+       }
+     },
+
+     /* Draws a table of TensorFlow Op statistics */
+     _showTopOpsTable: function(opsTableJson) {
+       var numRows = 0;
+       var opsTableBody = this.$.top_ops_table_content;
+       opsTableBody.innerHTML = '';
+       opsTableJson.rows.forEach(function(rowJson, index) {
+	 var row = document.createElement('tr');
+	 Polymer.dom(opsTableBody).appendChild(row);
+	 var cellTexts = [];
+	 cellTexts.push((rowJson.c[0].v*100).toFixed(2)+'%');
+	 cellTexts.push((rowJson.c[1].v*100).toFixed(2)+'%');
+	 cellTexts.push(rowJson.c[2].v);
+	 cellTexts.push(rowJson.c[3].v);
+	 cellTexts.push((rowJson.c[4].v).toFixed(2));
+	 cellTexts.forEach(function(cellText, index) {
+           var td = document.createElement('td');
+           Polymer.dom(row).appendChild(td);
+           Polymer.dom(td).appendChild(document.createTextNode(cellText));
+         });
+	 numRows += 1;
+       })
+       this.set('_top_ops_heading', 'Top ' + numRows + ' TensorFlow operations executed on TPU');
+       this.set('_show_top_ops_table', false);
+     },
+
+     /* Generates the HTML for a recommendation link */
+     _generateRecommendationHtml: function(tipsTableJson, cssClassName, wantedTipType, titleString) {
+       var content = '<p>&nbsp;</p>';
+       content += '<div class="' + cssClassName + '">';
+       content += '<font size="3"><b>' + titleString + ':</b></font>';
```

Comment:
Could we avoid the font tag and use CSS instead? The font tag had been introduced in the early days of HTML for formatting text, but has now been superseded by CSS formatting.